app/src/main/res/values-uk/strings.xml

```diff
@@ -63,5 +63,5 @@
     <string name="clock">Годинник</string>
     <string name="scroll">Прокрутка</string>
     <string name="set_sleep_time">Встановити час сну</string>
-    <string name="min_max">Min - %1$s \t\t Max - %2$s</string>
+    <string name="min_max">Мін. - %1$s \t\t Макс. - %2$s</string>
```

Comment:
Please remove dot symbol (.)

Reply:
Is it necessary? In Ukrainian grammatically correct way to write shortened words is with a dot at the end.

Reply:
does it changing the meaning of the word??


P.S. dot is also used in english for shortened words but i am more focusing on minimal app.

Reply:
Although it's not necessary, I want consistentency across different languages

Reply:
No. It's not a big deal if there won't be a dot.